index.html

```diff
@@ -0,0 +1,112 @@
+           <div class="main-program-content"> 
+            <h2>Main Program</h2>
+              <hr/>
+           </div>
+           <section class="main-program-card">
+              <div class="card">
+                <img src="images/lecture.png">
+                <p class="title">Lecture</p>
+                <p class="description">Listening to the panelist from all over africa</p>
+              </div>
+              <div class="card">
+                <img src="images/exhibition.png">
+                <p class="title">Exhibition</p>
+                <p class="description">Appreciation of various creation</p>
+              </div>
+              <div class="card">
+                <img src="images/forum.png">
+                <p class="title">Forum</p>
+                <p class="description">Have time to share your thoughts and opinion </p>
+              </div>
+              <div class="card">
+                <img src="images/workshop.png">
+                <p class="title">Workshop</p>
+                <p class="description">Working in groups different task provided</p>
+              </div>
+              <div class="card">
+                <img src="images/networking.png">
+                <p class="title">Networking</p>
+                <p class="description">Get opportunity to affliate around the attendies and creating friendships</p>
+              </div>
+           </section>
+              <button type="button" class="btn">JOIN YOUTH EMPOWERING SUMMIT 2023</button>
+              <a href="#" class="link-to-program"> SEE THE WHOLE PROGRAM </a>
+         </section>  
+         <section class="feature-section">
+               <div class="feature-speaker">
+                  <h3>Featured speakers</h3>
+                  <hr>
+               </div>
+               <section class="feature-card">
+               
+               </section>
+               <button  type="button" class="more-btn"> more </button>
+          </section>
+
+        </main>
+        <footer>
+          <section class="partners">
+              <div class="partner-head">
+                  <h3>Partners</h3>
+                  <hr>
+              </div>
+              <section class="partner-image">
+                 <img src="images/imbuto-foundation-logo.png" class ="footer-logo" alt="imbuto-foundation">
+                 <img src="images/YCA Main Logo.png" class ="footer-logo" alt="youth-connect-logo">
+                 <img src="images/google.png" class ="footer-logo" alt="google">
+                 <img src="images/airbnbmain.png" class ="footer-logo" alt="airbnb">
+                 <img src="images/lumen.png" class ="footer-logo" alt="lumen">
+              </section>
+          </section>
+          <section class="copyright">
+               <img src="images/YCA Main Logo.png" alt="youth-connect-logo">
+               <p> © Copyright YouthConnekt Africa 2023. All Rights Reserved.</p>
+              </section> 
```

Comment:
- [ ] You will notice there is a space between the logo and the text, kindly apply it to yours too.

## Your design
![image](https://user-images.githubusercontent.com/86200356/233290181-b41a3e18-f49e-4b9b-a8bc-cafc6459bcfb.png)

## Behance design
![image](https://user-images.githubusercontent.com/86200356/233290020-35dd1216-e9e5-4630-af6f-c4321f56eca8.png)

Reply:
Fixed.